java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S5852_java.html

```diff
@@ -22,10 +22,9 @@ <h2>Recommended Secure Coding Practices</h2>
   <li> If you have a non-possessive repetition <code>r*</code> or <code>r*?</code>, such that the regex <code>r</code> could produce different
   possible matches (of possibly different lengths) on the same input, the worst case matching time can be exponential. This can be the case if
   <code>r</code> contains optional parts, alternations or additional repetitions (but not if the repetition is written in such a way that there's only
-  one way to match it).
-    <p>When using JDK 9 or later an optimization applies when the repetition is greedy and the entire regex does not contain any back references. In that
```

Comment:
This should be reverted. During the writing of the description of this RSPEC, we did not find any way to obtain this behavior without adding it manually in the html.

Reply:
but then what if the text change ?! it's going to be a nightmare to follow!